test/active_record_shards_test.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require_relative 'helper'
+
+describe 'ActiveRecordShards' do
+  describe '.app_env' do
+    before do
+      if defined?(Rails) || ENV['RAILS_ENV'] || defined?(APP_ENV) || ENV['APP_ENV']
+        raise 'Tests in active_record_shards_test.rb will overwrite environment constants, please update them to avoid conflicts'
```

Comment:
In case the tests are moved around:
```suggestion
        raise "Tests in #{__FILE__} will overwrite environment constants, please update them to avoid conflicts"
```